src/main/resources/jenkins/branch/Messages.properties

```diff
@@ -54,7 +54,14 @@ OrganizationFolder.OrJoinN.First={0}, {1}
 OrganizationFolder.OrJoinN.Middle={0}, {1}
 OrganizationFolder.OrJoinN.Last={0} or {1}
 TaskNounUiTextProvider.TaskNoun=Scan
-NoTriggerOrganizationFolderProperty.PropertyMigration=The "Automatic branch project triggering" option has been replaced with a "Named branch" build strategy
+NoTriggerOrganizationFolderProperty.PropertyDeprecation=The "Automatic branch project triggering" option \
+    is deprecated. Use a "Named branch" build strategy from Basic Branch Build Strategies Plugin instead.
+NoTriggerOrganizationFolderProperty.PropertyMigrationWarning=The "Automatic branch project triggering" option \
+    for organization folders is deprecated and will be migrated to a "Named branch" build strategy from \
+    Basic Branch Build Strategies Plugin. If you were using "Automatic branch project triggering" to trigger \
+    builds for change requests (such as GitHub pull requests), then when you install version 1.1.0 or newer of \
+    Basic Branch Build Strategies Plugin, change requests will no longer be built. To preserve the previous behavior, \
+    you will need to manually reconfigure organization folders and add a "Change requests" build strategy.
```

Comment:
would it be possible to add a `href` here and a wiki page such that users can be given a fully worked example of what they need to do?

Reply:
We certainly could, with pictures, etc. For now we are discussing possible followups to this PR. If we end up disabling the migration and/or reverting the deprecation, all of this might be deleted, so I'll hold off until we have some clarity on next steps.

Reply:
446e87b disables the migration completely and makes the admin monitor no longer show up at all, so now this text will never be seen. I think it makes sense to keep it here for any future work that might end up reenabling the migration.